common/src/main/java/com/wynntils/wynn/model/discoveries/objects/DiscoveryInfo.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wynn.model.discoveries.objects;
+
+import com.wynntils.core.webapi.TerritoryManager;
+import com.wynntils.core.webapi.profiles.DiscoveryProfile;
+import com.wynntils.core.webapi.profiles.TerritoryProfile;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.mc.utils.ItemUtils;
+import com.wynntils.wynn.model.CharacterManager;
+import com.wynntils.wynn.utils.WynnUtils;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.world.item.ItemStack;
+
+public class DiscoveryInfo {
+    private final String name;
+    private final DiscoveryType type;
+    private final String description;
+    private final int minLevel;
+    private final TerritoryProfile guildTerritory;
+    private final boolean discovered;
+    private final List<String> requirements;
+    private List<Component> displayLore;
+
+    public DiscoveryInfo(DiscoveryProfile discoveryProfile) {
```

Comment:
I don't see why we can't merge this with `DiscoveryProfile`

Reply:
DiscoveryProfile is for GSON, merging it would be messy imo